pages/index.tsx

```diff
@@ -1,24 +1,141 @@
 import React from "react";
-import { GetServerSideProps } from "next";
-import { DataBreakdown, Table, Layout, TimeSeries } from "../components";
-import { Synth, getFakeSynths, getFakeTvl } from "../utils/mockData";
+import styled from "@emotion/styled";
 
-export const getServerSideProps: GetServerSideProps = async () => {
-  const tvlData = getFakeTvl();
-  const synthsData = getFakeSynths();
+import {
+  Layout,
+  Hero,
+  Card as UnstyledCard,
+  Value,
+  Table,
+} from "../components";
+import UnstyledInfoIcon from "../public/icons/info.svg";
+import { formatMillions, QUERIES } from "../utils";
 
-  return { props: { tvlData, synthsData } };
-};
-const IndexPage: React.FC<{ tvlData: TimeSeries; synthsData: Synth[] }> = ({
-  tvlData,
-  synthsData,
-}) => {
+const IndexPage: React.FC = () => {
   return (
     <Layout title="Umaverse">
-      <DataBreakdown data={tvlData} />
-      <Table data={synthsData} />
+      <Hero>
+        <Heading>
+          Explore the <span>UMA</span>verse
+        </Heading>
+        <Description>Another header lorem ipsum our projects</Description>
+        <CardWrapper>
+          <Card>
+            <InfoIcon />
+            <CardContent>
+              <CardHeading>
+                TVL <span>(Total Value Locked)</span>
+              </CardHeading>
+              <Value
+                value={22_850_000}
+                format={(v) => (
+                  <>
+                    ${formatMillions(v)}{" "}
+                    <span style={{ fontWeight: 400 }}>M</span>
+                  </>
+                )}
+              />
+            </CardContent>
+          </Card>
+          <Card>
+            <InfoIcon />
+            <CardContent>
+              <CardHeading>
+                TVM <span>(Total Value Minted)</span>
```

Comment:
For these headers, can you confirm they are the same size and font in the Figma? Looks slightly different

Reply:
I don't' think what's in parenthesis should be bolded

Reply:
I think for these headers it's better to have Total Value Locked (TVL) versus TVL (Total Value Locked)

Reply:
Clicking the UMA logo in the top left corner should take the user back to the main page of UMAverse instead of umaproject.org